### ISSUE TITLE
two class loss can not decline

### ISSUE THREAD

**Issue by lyxbyr** [VisualComputingInstitute/triplet-reid#87]
hello ,i train two class .set --batch_p = 2 --batch_k = 4,learing_rate has changed,loss can not decline, 
test result :mAP: 66.84% | top-1: 1.18% top-2: 1.78% | top-5: 1.78% | top-10: 2.37%
thanks

**Comment by Pandoro** [comment]
Please refrain from pinging unrelated people in the issues from now on.

I suspect your dataset is simply too easy. an mAP of 66.84% is not great, but it shows that the model learned something. If the hardest examples are clearly distinguishable, the loss should rather quickly go towards zero, or if be exactly zero if you do not use the soft-margin.

This is not a problem of our code though, but rather a machine learning issue for which we cannot give support here. My suggestion is that you gather a harder training set. I'm closing this issue though.
